homeassistant/components/mystrom/__init__.py

```diff
@@ -16,7 +16,7 @@
 from .const import DOMAIN
 from .models import MyStromData
 
-PLATFORMS_SWITCH = [Platform.SWITCH]
+PLATFORMS_SWITCH = [Platform.SWITCH, Platform.SENSOR]
```

Comment:
Can we rename `PLATFORMS_SWITCH` to `PLATFORMS_PLUGS` or something related instead

Reply:
Maybe, put propably not in this pr

Reply:
Please do the renaming in this PR, as it will improve the readability of the code. This constant is only used in two places in this file.